docs/modules/ROOT/pages/operations.adoc

```diff
@@ -339,12 +340,12 @@ The Bolt+routing feature on Neo4j drivers discovers a cluster member (drivers ha
 
 | `editorLint`
 | `false`
-| TODO
+| Enable/disable squiggly lines under text.
 | no
 
 | `useCypherThread`
 | `true`
-| TODO
+| Run cypher queries through a webworker to increase performance.
```

Comment:
| Run Cypher queries through a web worker (background thread) to increase performance.